--- data/BURCAT.THR.0	2025-12-07 16:54:04.118777419 +0100
+++ data/BURCAT.THR	2025-12-07 17:06:50.684828218 +0100
@@ -1692,7 +1692,7 @@
 7782-42-5
 C Carbon Solid Graphite Reference Element From B. McBride's Table based on TRC
 data. Max Lst Sq Error Cp @ 1300 K 0.65%.
-C<GR> REF ELEMENT T 3/10C  1.   0.   0.   0.S   200.000  6000.000  B  12.01070 1
+C[GR] REF ELEMENT T 3/10C  1.   0.   0.   0.S   200.000  6000.000  B  12.01070 1
  1.59828070E+00 1.43065097E-03-5.09435105E-07 8.64401302E-11-5.34349530E-15    2
 -7.45940284E+02-9.30332005E+00-3.03744539E-01 4.36036227E-03 1.98268825E-06    3
 -6.43472598E-09 2.99601320E-12-1.09458288E+02 1.08301475E+00 0.00000000E+00    4
@@ -3641,7 +3641,7 @@
 
 2683-96-7?
 CH2NO H2C*N=O RADICAL The G3B3 calculations show that this conformation is in
-resonance with the former conformation H2C*N=O <=> H2C=M-O* therefore the former
+resonance with the former conformation H2C*N=O [=] H2C=M-O* therefore the former
 polynomial should be used here. The present polynomial comes from the data of
 C. Melius BAC/MP4 calculations of 1987.  STATWT=2  SIGMA=2  IA=2.596  IB=3.041
 IC=5.0754   NU=3040,2952,1505,1304,1188,1111,983,897,846   HF298=53.52+/-2 kcal
@@ -7803,7 +7803,7 @@
 
 6912-06-7
 C2H3O Vinoxy radical (CH2=CHO*)  This radical is in resonance with the next
-radical CH2=CHO*<==>*CH2-CH=O which is probably preferred, therefore both
+radical CH2=CHO*[=]*CH2-CH=O which is probably preferred, therefore both
 radicals have the same polynomial   SIGMA=1  STATWT=2  Ia=1.2527  Ib=7.3645
 Ic=8.6172  Nu=3284,3174,2965,1572,1495,1416,1168,985,976,751,505,459
 HF298=12.75 kJ  HF0=20.189 kJ  REF=Burcat G3B3 calc {HF298=15.78+/-0.86 kJ 
