diff --git a/data/BURCAT.THR b/data/BURCAT.THR
index 28217ae..74ce40e 100644
--- a/data/BURCAT.THR
+++ b/data/BURCAT.THR
@@ -1611,7 +1611,7 @@ PBr3+  G3B3 calc  T05/15P  1.BR 3.E -1.   0.G   298.150  6000.000  B 270.68521 1
 7782-42-5
 C Carbon Solid Graphite Reference Element From B. McBride's Table based on TRC
 data. Max Lst Sq Error Cp @ 1300 K 0.65%.
-C<GR> REF ELEMENT T 3/10C  1.   0.   0.   0.S   200.000  6000.000  B  12.01070 1
+C[GR] REF ELEMENT T 3/10C  1.   0.   0.   0.S   200.000  6000.000  B  12.01070 1
  1.59828070E+00 1.43065097E-03-5.09435105E-07 8.64401302E-11-5.34349530E-15    2
 -7.45940284E+02-9.30332005E+00-3.03744539E-01 4.36036227E-03 1.98268825E-06    3
 -6.43472598E-09 2.99601320E-12-1.09458288E+02 1.08301475E+00 0.00000000E+00    4
@@ -3488,7 +3488,7 @@ CH2NO  H2C=NO*    T06/08C  1.H  2.N  1.O  1.G   200.000  6000.000  B  44.03272 1
 
 2683-96-7?
 CH2NO H2C*N=O RADICAL The G3B3 calculations show that this conformation is in
-resonance with the former conformation H2C*N=O <=> H2C=M-O* therefore the former
+resonance with the former conformation H2C*N=O [=] H2C=M-O* therefore the former
 polynomial should be used here. The present polynomial comes from the data of
 C. Melius BAC/MP4 calculations of 1987.  STATWT=2  SIGMA=2  IA=2.596  IB=3.041
 IC=5.0754   NU=3040,2952,1505,1304,1188,1111,983,897,846   HF298=53.52+/-2 kcal
@@ -7120,7 +7120,7 @@ C2H3O-  CH3CO-    T 6/13C  2.H  3.O  1.   0.G   200.000  6000.000  B  43.04462 1
 
 6912-06-7
 C2H3O Vinyl oxy radical (CH2=CHO*)  This radical is in resonance with the next
-radical CH2=CHO*<==>*CH2-CH=O which is probably preferred, therefore both
+radical CH2=CHO* [=] *CH2-CH=O which is probably preferred, therefore both
 radicals have the same polynomial   SIGMA=1  STATWT=2  Ia=1.2527  Ib=7.3645
 Ic=8.6172  Nu=3284,3174,2965,1572,1495,1416,1168,985,976,751,505,459
 HF298=12.75 kJ  HF0=20.189 kJ  REF=Burcat G3B3 calc  Max Lst Sq Error Cp @
@@ -10806,7 +10806,7 @@ HF298=107.755+/-8. kJ  REF=Burcat G3B3  thermal electron  Max Lst Sq Error Cp @
 1300 K 0.51%.
 The electron is taken into acount but appears only in the therm.dat polynomial
 file and in the 9 term (NEWNASA) polynomial.
-C3H9PO3+ (CH3O)3P+T 7/16 WARNING Missing E  G   298.150  6000.000  B 124.07497 1
+C3H9PO3+ (CH3O)3P+T 7/16C  3.H  9.P  1.O  3.E -1.G   298.150  6000.000  B 124.07497 1
  1.47566876E+01 2.58667672E-02-9.15297579E-06 1.46249830E-09-8.69184379E-14    2
  6.47471610E+03-4.41714462E+01 9.05909479E+00 2.40681583E-02 2.83884688E-05    3
 -4.76721278E-08 1.90024648E-11 9.02357838E+03-1.01801200E+01 1.29598738E+04    4
@@ -10820,7 +10820,7 @@ IB=53.3754  IC=77.2109  [Ir(CH3O)=3.7003  ROSYM=1  V(3)=900. cm-1]x3  Nu=3150,
 REF=Burcat G3B3  Max Lst Sq Error Cp @ 6000 K 0.54%.
 The electron is taken into acount but appears only in the therm.dat polynomial
 file and in the 9 term (NEWNASA) polynomial.
-C3H9PO3-          T 7/16 WARNING! Missing E G   298.150  6000.000  B 124.07607 1
+C3H9PO3-          T 7/16C  3.H  9.P  1.O  3.E  1.G   298.150  6000.000  B 124.07607 1
  1.53860900E+01 2.61911320E-02-9.53433742E-06 1.54032420E-09-9.19903877E-14    2
 -2.85880823E+05-4.19674496E+01 1.01910843E+01 2.63281591E-02 1.64582413E-05    3
 -3.20572869E-08 1.25710205E-11-2.83551723E+05-1.11814873E+01-2.79255057E+05    4
@@ -31864,4 +31864,4 @@ A. Burcat paper cited as REF.
 polyciclyc compounds.
 
 
-   
\ No newline at end of file
+   
